problemtools/problem2html.py

```diff
@@ -38,7 +38,12 @@ def convert(problem, options=None):
 
         # Setup parser and renderer etc
 
-        tex = plasTeX.TeX.TeX(myfile=texfile)
+        try:
+            # try the new version of plasTeX (in version 3.0, the argument is named "file")
+            tex = plasTeX.TeX.TeX(file=texfile)
```

Comment:
This looks rather fugly to me. Could you peek at plasTeX.__version__ instead of the try and bare except dance?

Reply:
Your suggestion was my first idea as well, but unfortunately right now the plasTeX source has the incorrect `__version__`, which I have reported here: https://github.com/plastex/plastex/issues/308

So for now (until that is fixed) there's no good way to tell which version of plasTeX you have without either doing this method (try/except) or trying some inspection on the name of the arguments, which also felt ugly.

Reply:
@ghamerly PlasTeX were rather responsive, and your issue seems to be fixed already ;)

Reply:
Good; I'll push a better fix now that the version in plasTeX is fixed.